src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -215,7 +220,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('fragment_renderer.php');
         $loader->load('error_renderer.php');
 
-        if (interface_exists(PsrEventDispatcherInterface::class)) {
+        if ($container::willBeAvailable('psr/event-dispatcher', PsrEventDispatcherInterface::class, ['symfony/framework-bundle'])) {
```

Comment:
As this is a static method, I'd rather use `ContainerBuilder::willBeAvailable` to make the usage more clear

Reply:
thanks for the review, updated!